models/refugee.py

```diff
@@ -92,30 +100,34 @@ def __sanitise_starting_date(self, starting_date: date):
             raise self.InvalidStartingDateException()
         return starting_date
 
+    @property
+    def camp(self):
+        from models.camp import Camp
+        return self.find_referred_by(referrer_type=Camp)
 
     def __str__(self):
-        return f"Refugee family {self.firstname} {self.lastname} located in {self.camp}.\n"\
+        return f"Refugee family {self.firstname} {self.lastname} located in {self.camp}.\n" \
                f"Number of Family Member: {self.num_of_family_member}\n" \
                f"Camp: {self.camp}\n" \
                f"Creation Date: {self.starting_date}\n" \
-               f"Medical Condition: {self.medical_condition_type}\n"\
-
+               f"Medical Condition: {self.medical_condition_type}\n"
 
     class InvalidNumOfFamilyMemberException(Exception):
         """
         Raise exception when number of family member entered.
```

Comment:
Could this be corrected to: "Raise exception when the number of family member(s) entered is invalid."